examples/csharp/aws/package.json

```diff
@@ -8,4 +8,4 @@
     "build": "cdktf get",
     "synth": "cdktf synth"
   }
-}
\ No newline at end of file
+}
```

Comment:
Did prettier do this? Makes this PR a bit harder to review, as lots of files seem changed although they just where cleaned up 🙈
(just a nit, feel free to ignore)